presto-product-tests/src/main/java/io/prestosql/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -221,6 +221,15 @@ public void testAccessControlSetTableAuthorization()
         bobExecutor.executeQuery(format("ALTER TABLE %s SET AUTHORIZATION alice", tableName));
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testAccessControlSetViewAuthorization()
+    {
+        assertThat(() -> bobExecutor.executeQuery(format("ALTER VIEW %s SET AUTHORIZATION bob", viewName)))
+                .failsWithMessage(format("Access Denied: Cannot set authorization for view default.%s to USER bob", viewName));
+        aliceExecutor.executeQuery(format("ALTER VIEW %s SET AUTHORIZATION bob", viewName));
```

Comment:
There is missing assertion that would prove that authorization was actually set. Below line could pass from other reasons too.